src/__tests__/cogniteClient.spec.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright 2019 Cognite AS
 
 import MockAdapter from 'axios-mock-adapter';
+import { CogniteClient } from '..';
```

Comment:
Import this in the line below instead

Reply:
The linter keeps moving it to this position. Should I ignore the linter, or will that cause failed CI tests?

Reply:
no, linter is our king. just obey it :)
but that's weird tho, might be a bug

Reply:
@f1cognite doing `import { CogniteClient, POPUP, REDIRECT } from '../cogniteClient';` gives error, if that's what you were thinking about doing.

Reply:
`import CogniteClient, { POPUP, REDIRECT } from '../cogniteClient';` fixes the problem, done

Reply:
`import CogniteClient, { POPUP, REDIRECT } from '../cogniteClient';`